### PR TITLE
Truncate long testcase comments before updating.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
PTAL. seeing some exceptions with entity sizes on internal.

File: src/python/datastore/data_types.py [hunk 1]
```diff
@@ -99,6 +99,12 @@
 # We store at most 3 stacktraces per Testcase entity (original, second, latest).
 STACKTRACE_LENGTH_LIMIT = ENTITY_SIZE_LIMIT // 3
 
+# Maximum size allowed for testcase comments.
+# 1MiB (maximum Datastore entity size) - ENTITY_SIZE_LIMIT (our limited entity
+# size with breathing room), divided by 2 to leave room for other things in the
+# entity. This is around 74KB.
+TESTCASE_COMMENTS_LENGTH_LIMIT = (1024 * 1024 - STACKTRACE_LENGTH_LIMIT) // 2
```
[the remark this answers is not in the document]
Comment (oliverchang) [reply src/python/datastore/data_types.py]: Fixed! I actually fixed the comment to use ENTITY_SIZE_LIMIT, but failed to change it in the actual code.

**Comment by oliverchang** [comment]
Thanks for fixing the android issue!

**Comment by inferno-chromium** [comment]
> Thanks for fixing the android issue!

Still speculating based on analysis on few failures, hope it works.

**Pull request merged**